elastic/kibana
# 6425 correlations in insights

## Summary

This PR adds the correlations details section, according to the following issue:
https://github.com/elastic/security-team/issues/6425

It looks like this:

![image](https://github.com/elastic/kibana/assets/11671118/dcd64cbe-a861-4f51-87ec-6d9be101e8e6)

Testing:

Easiest way to generate data for this section I believe is to use the following command

yarn test:generate --anc 5 --related 5 --relAlerts 5

You should enable the default rule set for the alerts to pop up.

* add xpack.securitySolution.enableExperimental: ['securityFlyoutEnabled'] to the kibana.json file
* run yarn es snapshot --license trial, yarn test:generate and yarn start --no-base-path
* go to the Alerts page, and click on the expand detail button on any row of the table
* expand alert
* go to "insights" tab on the left side
* go to "correlations" sub-tab

### Checklist

Delete any items that are not applicable to this PR.
